enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -96,6 +98,19 @@ func (c *podmanCommandContainer) getPodmanRunArgs(workDir string) []string {
 	if c.options.CapAdd != "" {
 		args = append(args, "--cap-add="+c.options.CapAdd)
 	}
+	for _, device := range c.options.Devices {
+		deviceSpecs := make([]string, 0)
+		if device.PathOnHost != "" {
+			deviceSpecs = append(deviceSpecs, device.PathOnHost)
+		}
+		if device.PathInContainer != "" {
+			deviceSpecs = append(deviceSpecs, device.PathInContainer)
+		}
```

Comment:
If PathInContainer is unspecified, we should set it to match the path on host, otherwise if cgroup permissions are specified, the flag will wind up as `/host/path:cgroupperms` and the `cgroupperms` will be parsed as the container path:

```suggestion
		if device.PathInContainer != "" {
			deviceSpecs = append(deviceSpecs, device.PathInContainer)
		} else {
			deviceSpecs = append(deviceSpecs, device.PathOnHost)
		}
```

Reply:
Before implementation, I have took a look at podman testing suite: [relevant link](https://github.com/containers/podman/blob/418ab2e5e154cb0df13f2e901cf4aea31a5cd0a9/test/e2e/run_device_test.go#L63)
It is my understanding that without PathInContainer, podman automatically maps it to same path as PathInHost and that writting CGroupPerms at second place (instead of PathInContainer) is supported. 

Please let me know if that works or should I implement the execution flow accordingly to your suggestion :)

Reply:
Thank you for looking more deeply into that! LGTM as-is